src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,119 @@
+package duke;
+
+/**
+ * The class with deals with user interaction.
+ */
+public class Ui {
+    public static final String LINE_BREAK = "____________________________________________________________";
+
+    /**
+     * Prints a welcome message when the program starts.
+     */
+    public void welcome() {
```

Comment:
Perhaps this should be:
```suggestion
    public void showWelcome() {
```
instead? According to [Java coding standards](https://se-education.org/guides/conventions/java/intermediate.html), names representing methods must be verbs.